__tests__/Logger-test.js

```diff
@@ -9,43 +9,49 @@ const Log = new Logger({
 });
 
 test('Test Log.info()', () => {
-    Log.info('Test', true);
+    Log.info('Test1', false);
+    expect(mockServerLoggingCallback).toHaveBeenCalledTimes(0);
+    Log.info('Test2', true);
     expect(mockServerLoggingCallback).toHaveBeenCalled();
-    expect(mockServerLoggingCallback).toHaveBeenCalledWith({
-        parameters: {},
-        api_setCookie: false,
-        message: '[info] Test',
-    });
+    expect(mockServerLoggingCallback).toHaveBeenCalledWith(
+        expect.objectContaining({
+            api_setCookie: false,
+            logPacket: expect.any(String),
```

Comment:
These tests are not a bit shitty, since the logPacket contains a timestamp, I don't know how to make this test a bit better (like check its including the actual log lines).

Reply:
jest functions can have their arguments inspected so that might be one way?

e.g. `mockServerLoggingCallback.mock.calls` should return an array of arrays representing each calls arguments. So to get the first argument of the first call you'd do

```js
mockServerLoggingCallback.mock.calls[0][0]
```

Reply:
And then we can do like `expect(args).toEqual({something: 'blah'})`